modules/machineset-delete-policy.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * machine_management/manually-scaling-machineset.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="machineset-delete-policy_{context}"]
+= The MachineSet delete policy
+
+`Random`, `Newest`, and `Oldest` are the three supported options. The default is `Random`, meaning  that random machines are chosen and deleted when scaling MachineSets down. The delete policy can be set according to the use case by modifying the particular MachineSet:
```

Comment:
In the first sentence in this line, you could change "the three supported options" to "the three supported deletion options" or similar, to be explicit about what is being referred to.

Reply:
You could change "MachineSets" to "machine sets" in line with the new API object reference guidelines or refer to the resource directly (e.g. "when scaling the `MachineSet` resources down"). There are a number of other instances of "MachineSets" and "MachineSet" in this assembly. Please consider updating all of them. I will not point them all out in this review.

Reply:
Thanks, Paul! I'll go with "the three supported deletion options".

Since these sections are planned to be updated by another assigned writer to follow the new API object reference guidelines (and since no text changes were made here, just module separation), I'm going to refrain from bulk updating those references in this PR.